app/src/main/java/com/example/rwothoromo/developers/model/GithubUsers.java

```diff
@@ -0,0 +1,64 @@
+package com.example.rwothoromo.developers.model;
```

Comment:
This is better named as `GithubUser` according to popular conventions since it represents a single entity.

Reply:
Oh yes, that makes more sense just that the PivotalTracker story had it in the plural.